src/components/com_tjcertificate/site/models/forms/filter_certificates.xml

```diff
@@ -19,6 +20,7 @@
 			clientByUser="1"
 			onchange="this.form.submit();" />
 
+
```

Comment:
You can remove this file from the MR as there is no notable changes

Reply:
done